test.sh

```diff
@@ -13,6 +13,8 @@ if ! which golint > /dev/null ; then
   exit 1
 fi
 fgt golint ./...
-go test ./...
+go test $BUILD_FLAGS ./...
 go list -f '{{if len .TestGoFiles}}"go test -coverprofile={{.Dir}}/.coverprofile {{.ImportPath}}"{{end}}' ./... | xargs -i sh -c {}
 gover . coverprofile.txt
+go get $BUILD_FLAGS -d ./...
```

Comment:
These two commands (the go get and go build) are now redundant. For Go projects, Travis already runs `go get -t -v ./...` during its `install` phase. And the `go test $BUILD_FLAGS ./...` above automatically does a go build for that directory. I'd recommend removing this line and the one below.